content/en/blog/2023/speeding-up-container-image-builds-in-tekton-pipelines/index.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Speeding Up Container Image Builds in Tekton Pipelines
+linkTitle: Speeding Up Container Image Builds in Tekton Pipelines
+date: 2023-11-2
```

Comment:
The date isn't showing up properly in the preview, I think it requires zero-padding the date

![image](https://github.com/tektoncd/website/assets/2829095/e9705b3f-2f6d-447d-84ea-2d8ec9dd3c1b)
![image](https://github.com/tektoncd/website/assets/2829095/9b1ce9c2-7e90-4c1f-b579-3ac83c913598)

```suggestion
date: 2023-11-02
```